index.html

```diff
@@ -5,16 +5,158 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
-    <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+    <link href="css/main.css" rel="stylesheet" type="text/css">
+    <link href="css/grid.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Anton&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ovo&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rokkitt&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Vast+Shadow&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/0729575a56.js" crossorigin="anonymous"></script>
+    <title>
+        Steve Jobs
+    </title>
 </head>
+<!-- <div class="hamIcon" onclick="toggleClass()">
+            <h1> &#9776; </h1>
+        </div>-->
 
 <body>
+    <header class="header_customize">
+        <img class="logo_img" src="steve_jj.jpeg">
+        <ul class="ul_header">
+            <li>Steve Jobs</li>
+            <br>
+            <li>co-founded Apple Computers</li>
+        </ul>
+        <ul class="ul_header_nav">
+            <li><a href="#home">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#Blog">Blog</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#login"><i class="fas fa-user">Login</i></a></li>
+        </ul>
+    </header>
+    <main>
+        <section class="hero_image section_one">
+            <h1 class="font_anton">
+                SUCCESS
+            </h1>
+            <button class="button col-2">
+                Book Now
+            </button>
+        </section>
+        <section>
```

Comment:
If you split the text part below and the image part (with the product image) into 2 different sections, you could add a `container` on your text section to 'contain' the text to the center of the screen on large screens. At the moment, the text goes all the way to the edges which looks a bit much on desktop.